src/CMakeLists.txt

```diff
@@ -292,6 +292,8 @@ if(WIN32)
 		set(VORBIS_DLL "" CACHE FILEPATH "Path to libvorbis.dll for installation (optional)")
 		set(VORBISFILE_DLL "" CACHE FILEPATH "Path to libvorbisfile.dll for installation (optional)")
 	endif()
+
+	set(LUA_DLL "" CACHE FILEPATH "Path to lua51.dll for installation (optional)")
```

Comment:
Done.

Well, if you use a custom build script, the name may differ, but using the [official build instructions](http://luajit.org/install.html), its called `lua51.dll`